va/va.h

```diff
@@ -1401,6 +1408,21 @@ typedef union _VAConfigAttribValContextPriority {
     uint32_t value;
 } VAConfigAttribValContextPriority;
 
+/** brief Attribute value VAConfigAttribEncPerMBControl */
+typedef union VAConfigAttribValEncPerMBControl {
+    struct {
+        /** \brief whether to support dela qp per block */
+        uint32_t delta_qp_support         : 1;
+        /** \brief supported size of delta qp block */
+        uint32_t log2_delta_qp_block_size : 4;
+        /** \brief size of delta qp in bytes*/
+        uint32_t delta_qp_size            : 3;
```

Comment:
is there better name?  it is confusing , whole buffer size of per qp value size

Reply:
Ok,  I'll get a better name for this field.